requirements.txt

```diff
@@ -1 +1,2 @@
 git+git://github.com/canonical/operator
+jinja2
```

Comment:
Yes, completely. Not only does the new Grafana dashboard use Jinja as a standard template language (which is in no way unusual -- Ansible, Django, and a number of other very large projects have put a lot of work into it because template strings are not adequate), but the machine charms absolutely use Jinja for their dashboards.